### DISCUSSION TITLE
被 AI 主導的現代戰爭開打中，自動化武器會是跨過人性疆界的最後一個臨界點嗎？

### DISCUSSION THREAD

**Discussion by yi-1111** [1101-GS4719-C/Python-Community-GS4719-C#505] [閒聊八卦]
自駕車、自動無人工廠看起來為人類帶來光明的未來願景。透過電腦運算，自駕車可以避開障礙物、確保安全；無人工廠可以提升生產效能與產品良率，讓產線上工作者的工作能升級。那麼讓戰爭攻擊可以更加精準與「人道」的自動化武器，是個好的發明嗎？

隨著烏俄開打，AI是否利用於戰爭中，那未來它定位該何去何從?
歡迎大家發表想法
https://buzzorange.com/techorange/2022/03/02/hoozbook-army-of-none/

**Comment by sean1325418** [comment]
以後上戰場的可能都是機器人，人類只要在後方操控就好

**Comment by ghost** [comment]
不太可能，真全部機械上陣，一個癱瘓就直接被直搗黃龍了

**Comment by hapiwangy** [comment]
在戰爭裡面討倫人道，我認為是不夠成熟的想法。不過也很好奇ＡＩ是否真的能夠改進（？）戰爭的型態

**Comment by PoooYen** [comment]
我覺得可以應用在戰爭裡面減少人員的傷亡，就讓機器人去打仗，其實現在已經有無人機投入作戰的大量範例了，只是現在還是半自動操控而已

**Comment by Jimmy911218** [comment]
想不到AI盡然也變成了人奪權的工具，人的野心還真無窮啊!
